packages/next/build/generate-build-id.ts

```diff
@@ -2,7 +2,11 @@ export async function generateBuildId (generate: () => string|null, fallback: ()
   let buildId = await generate()
   // If there's no buildId defined we'll fall back
   if (buildId === null) {
-    buildId = fallback()
+    // We also create a new buildId if it starts with `ad` to avoid false 
+    // positives with ad blockers
+    while (!buildId || buildId.startsWith('ad')) {
```

Comment:
Do we think it would trigger adblock as well if the hash has `ad` anywhere else than the beginning of the string?

Reply:
I've created a PR which solves this: https://github.com/zeit/next.js/pull/6854